assets/css/screen-detail.scss

```diff
@@ -75,7 +75,7 @@
 
   &--bus_eink_v2,
   &--gl_eink_v2 {
-    height: 480px;
+    height: 503px;
```

Comment:
Are these heights changed to keep the vertical scrollbar hidden?

Reply:
Sorta, but also no -- I changed these values and the heights for `.screen-detail__iframe` to remove the vertical scrollbar, but it turns out that I only needed to change the values for the iframe and not the iframe-container. So I've reverted these ones, because it helps remove unnecessary padding.